README.md

```diff
@@ -1,3 +1,20 @@
 # kodim-starter-kits
 
-Project templates for create-kodim-app.
\ No newline at end of file
+Project templates for create-kodim-app.
+
+## Usage
+
+```bash
+npm init kodim-app@latest <project-name> [kit-name]
+npm install # only for React kits
+npx vite
```

Comment:
Proč ne `npm start` jako tomu bylo dříve?

Reply:
> Proč ne `npm start` jako tomu bylo dříve?

`npm start` vyžaduje `package.json`. `npx vite` funguje i pro projekty bez `package.json`, takže se po celou akademii od začátku (od HTML+CSS) až do konce může používat pořád stejný příkaz `npx vite`. A nemusíme řešit, že se `npx serve` chová jinak – všechno bude od začátku servírované přes Vite (jehož server neodstraňuje přípony souborů a neodstraňuje query parametry jako `npx serve`).

Reply:
Pouštět `npx vite` v projektech, které mají `package.json` mi nepřijde jako dobrý nápad.

1. Naučit se pouštět `npm start` je pro účastnice užitečnější do života. Univerzálnější příkaz.
2. `vite` je hodně komplexní. Vystavujeme se obrovskému riziku, že vyjde novější verze, která se bude chovat až moc jinak. (Požívat místo `npx serve` mi přijde ještě ukočírovatelné.) `npx vite` je náročnější na údržbu. Musíme sledovat releasy major verzí. Můžou nastat problémy s cachováním `npx` (`kodim-app@latest`).
3. Pokud účastnice pustí `npx vite` v projektu, kde měl být `package.json` a není, obávám se, že se budou dít velmi matoucí věci.
4. `npx vite` bude v projektech s `package.json` jistě dost matoucí pro kouče, mentory a lektory. Je to až moc specifický setup pro pouštění projektů.
6. Zároveň také bude větší průšvih, pokud na nějakém projektu zapomeneme Webpack. S `npm start` by si toho účastnice teoreticky nemusely ani všimnout.
7. Pokud se časem opět rozhodneme změnit bundler, s `npx vite` všude v materiálech budeme mít více práce.

Netroufám si to rozseknout ve dvou. Nekonzultoval jsi to náhodou ještě s někým? Třeba s @lrolecek?